apps/dcos_l4lb/src/dcos_l4lb_ipvs_mgr.erl

```diff
@@ -268,6 +279,24 @@ handle_get_dests(Service, Namespace, #state{netns = NetnsMap, family = Family})
     [proplists:get_value(dest, MaybeDest) || #netlink{msg = #new_dest{request = MaybeDest}} <- Replies,
         proplists:is_defined(dest, MaybeDest)].
 
+-spec(handle_mod_dest(ServiceIP :: inet:ip_address(), ServicePort :: inet:port_number(),
+                      DestIP :: inet:ip_address(), DestPort :: inet:port_number(),
+                      Protocol :: protocol(), Namespace :: term(), State :: state(),
+                      Weight :: backend_weight()) -> ok | error).
+handle_mod_dest(ServiceIP, ServicePort, DestIP, DestPort, Protocol,
+                Namespace, #state{netns = NetnsMap, family = Family}, Weight) ->
+    Pid = maps:get(Namespace, NetnsMap),
+    Protocol1 = netlink_codec:protocol_to_int(Protocol),
+    Service = ip_to_address(ServiceIP) ++ [{port, ServicePort}, {protocol, Protocol1}],
+    Base = [{fwd_method, ?IP_VS_CONN_F_MASQ}, {weight, Weight}, {u_threshold, 0}, {l_threshold, 0}],
+    Dest = [{port, DestPort}] ++ Base ++ ip_to_address(DestIP),
+    lager:info("Modifying backend ~p for service ~p~n", [{DestIP, DestPort, Weight}, Service]),
+    Msg = #set_dest{request = [{dest, Dest}, {service, Service}]},
```

Comment:
Damn, it took me a while to find the `set_dest` record 😄 I didn't knew we had forked the `gen_netlink` module...

Reply:
Yea :(, I wish we had renamed it after it looked like upstream wasn't going to maintain it.